Tests/Imagine/Filter/Loader/FloatToIntCastByRoundUpscaleFilterLoaderTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Liip\ImagineBundle\Tests\Filter;
+
+use Imagine\Gd\Image;
+use Imagine\Gd\Imagine;
+use Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader;
+use Liip\ImagineBundle\Tests\AbstractTest;
+
+/**
+ * @covers Liip\ImagineBundle\Imagine\Filter\Loader\UpscaleFilterLoader
+ *
+ * Due to int casting in Imagine\Image\Box which can lead to wrong pixel
+ * numbers ( e.g. float(201) casted to int(200) ). Solved by round the
```

Comment:
This is what is rely strange. By executing the test on master while var_dump'ing ratio and the arguments passed to the constructor i get the following:

    var_dump($ratio, $origHeight * $ratio);
    float(4.02)
    float(201)